tests/test_sequence.py

```diff
@@ -922,76 +926,54 @@ def test_sequence(reg, device, patch_plt_show):
     assert str(seq) == str(seq_)
 
 
-def test_config_slm_mask(device):
-    reg_s = Register({"q0": (0, 0), "q1": (10, 10), "q2": (-10, -10)})
-    seq_s = Sequence(reg_s, device)
-
+@pytest.mark.parametrize("qubit_ids", [["q0", "q1", "q2"], [0, 1, 2]])
+def test_config_slm_mask(qubit_ids, device):
+    reg: Register | MappableRegister
```

Comment:
What is mypy complaining about without this?

Reply:
Oh it's just a simple rewriting of the test. I found it was better to write it with a parametrize call.

I implemented this when I thought I could also modify the slm. With my changes it was making the initial test super long and I found it necessary to synthetize it. Now these changes are unnecessary but I find it nicer this way. If you don't find so I will cancel these changes.

Reply:
No, that's ok, I was just wondering